django/db/models/functions/math.py

```diff
@@ -0,0 +1,122 @@
+from django.db.models import Func, Transform
+from django.db.models.fields import FloatField
```

Comment:
`FloatField` can be imported from `django.db.models`, so merge with the line above.

Reply:
Hi, I tried:
from django.db.models import Func, Transform, .fields.FloatField
But it doesn't work.

Reply:
I meant: `from django.db.models import FloatField, Func, Transform`

Reply:
Thanks!